### DISCUSSION TITLE
Unable to load games from tape

### DISCUSSION THREAD

**Discussion by BulletMag** [UzixLS/zx-sizif-512#11] [Q&A]
After building the unit I was happy to see it powered on and displayed fine, 
I now have purchased a tapduino to load games however when I either use the "tape loader" option or type Load "" the border does not flash as I remember and remains solid white with no sound...
If I plug the tapduino into speakers I hear the normal loading as I would expect. 
I have also installed the Bluetooth module and paired with my phone, this too appears to not load anything and again the screen stays solid white...

What am I doing wrong ?

**Comment by UzixLS** [comment]
Check soldering of following elements: R47, C51, R57, R5, D11, D15, U9, U6 (pin 86). U9 should have "U" suffix (74HCU04). Also check that turbo mode is disabled (CPU Freq = 3.5MHz)

**Reply by BulletMag** [reply]
Checked all of the above ... No joy however I did find a bridge between two pins on U6 and after removing this I got 128k sound (this was a secondary issue but I thought I had fake AY chips) 
I changed one of the diodes as it looked cracked (it wasn't after I inspected it) but still the unit fails to load from Bluetooth or cable.
DIVMMC loading works so I can load games but it's just the memories of manually loading and waiting that appeals.

Oh and when esxdos loads RTC reports an error

And one more thing how do I get settings to save as if I use the 'Save Settings' in the advanced menu it just says error and fails to save if I bridge JP14 the unit crashes so I'm at a loss.

**Reply by UzixLS** [reply]
> Oh and when esxdos loads RTC reports an error

It's ok for esxdos


> And one more thing how do I get settings to save as if I use the 'Save Settings' in the advanced menu it just says error and fails to save if I bridge JP14 the unit crashes so I'm at a loss.

Did you mean JP4? Probably there is short between U8 pin 31 and something else (+5 or etc). Also currently only SST39SF0x0 supported to save settings.
